securedrop/journalist.py

```diff
@@ -321,7 +321,7 @@ def index():
         else:
             unstarred.append(source)
         source.num_unread = len(
-            Submission.query.filter(Submission.source_id == source.id, Submission.downloaded == False).all())
+            Submission.query.filter(Submission.source_id == source.id, Submission.downloaded is False).all())
```

Comment:
Do we really want to use `is` instead of `==`? I understand that `is` will work, but that seems to be a special case that only works for booleans, small integers, etc. in CPython. Isn't `is` better used for comparing object identity, instead of value?